src/models/article.models.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (sequelize, Sequelize) => {
+  const Article = sequelize.define('article', {
+    slug: {
+      primaryKey: true,
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    title: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    description: {
+      type: Sequelize.STRING,
+    },
+    body: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    favorited: {
+      type: Sequelize.BOOLEAN,
+    },
+    favoritesCount: {
+      type: Sequelize.INTEGER,
+    },
+  });
+  return Article;
+};
```

Comment:
```suggestion
const { Model } = require('sequelize');

class Article extends Model {}

/**
 * Initialize article model.
 *
 * @type {typeof Article}
 *
 * @param {import('sequelize').Sequelize} sequelize Sequelize class.
 * @param {import('sequelize')}           dataTypes Sequelize data types.
 *
 * @return {Article} Article model.
 */
module.exports = (sequelize, dataTypes) => {
  Article.init(
    {
      articleId: {
        allowNull: false,
        defaultValue: dataTypes.literal('uuid_generate_v4()'),
        primaryKey: true,
        type: dataTypes.UUID,
      },
      title: {
        allowNull: false,
        type: dataTypes.STRING,
      },
      description: {
        type: dataTypes.STRING,
      },
      body: {
        allowNull: false,
        type: dataTypes.STRING,
      },
      favorited: {
        defaultValue: false,
        type: dataTypes.BOOLEAN,
      },
      favoritesCount: {
        defaultValue: 0,
        type: dataTypes.INTEGER,
      },
    },
    { sequelize, modelName: 'Article', tableName: 'articles' }
  );
  return Article;
};

```

also please rename this file to `article.js `